components/service-public-section/index.tsx

```diff
@@ -0,0 +1,76 @@
+import { ReactNode } from 'react';
+import NonRenseigne from '#components/non-renseigne';
+import { DataSection } from '#components/section/data-section';
+import { TwoColumnTable } from '#components/table/simple';
+import { EAdministration } from '#models/administrations/EAdministration';
+import { IAPINotRespondingError } from '#models/api-not-responding';
+import { IUniteLegale } from '#models/index';
+import { IServicePublic } from '#models/service-public';
+
+export default function ServicePublicSection({
+  servicePublic,
+  uniteLegale,
+}: {
+  uniteLegale: IUniteLegale;
+  servicePublic: IServicePublic | IAPINotRespondingError;
+}) {
+  return (
+    <>
+      <DataSection
+        title={'Service public'}
```

Comment:
I hesitate between `Annuaire du service public` which is the name of the source and "service public" which is coherent with "collectivité territoriale"

Reply:
I think we use the name of the data instead of the API almost everywhere else. Let's keep it that way.